### DISCUSSION TITLE
Can't see the export module

### DISCUSSION THREAD

**Discussion by olivermontalvanm** [kimai/kimai#3394] [Questions & Answers]
I am logged in as system admin, however I cannot see the export module... When I try to manually type the /export URL, it brings up a restricted page (I don't have access to export).

Any ideas?...

**Comment by kevinpapst** [comment]
You locked yourself out:
https://www.kimai.org/documentation/permissions.html#existing-permissions

**Comment by olivermontalvanm** [comment]
I turned all permissions ON for my role and still cannot see it...
